tutorials/calibration_flow/AWQ.py

```diff
@@ -0,0 +1,145 @@
+import torch
```

Comment:
this folder is for examples for developers actually, can you move this to `torchao/prototype/awq/example.py`